web/angular-wallet/src/app/layout/components/burst-fee-selector/burst-fee-selector.component.html

```diff
@@ -10,8 +10,10 @@
     </a>
   </div>
   <div class="__fees">
-    {{'minimum_fee' | i18n}}:<em (click)="feeNQT=convertFeeToBurst(fees.minimum)">{{ convertFeeToBurst(fees.minimum) }}</em>
-    {{'cheap_fee' | i18n}}:<em (click)="feeNQT=convertFeeToBurst(fees.cheap)">{{ convertFeeToBurst(fees.cheap) }}</em>
+    {{'minimum_fee' | i18n}}: <em
```

Comment:
line breaks are somehow unusual...

Reply:
I made by similar style as standard_fee and priority_fee